src/main/java/org/hydev/mcpm/client/arguments/CommandsFactory.java

```diff
@@ -31,14 +26,15 @@ private CommandsFactory() { }
      */
     public static List<CommandParser> baseParsers() {
         var echoController = new EchoController();
+        var exportPluginsController = new ExportPluginsController(null); // do I have to instantiate the entire thing ??
```

Comment:
Yep, you do 😰

Reply:
I should mention that `CommandsFactory` is just a suggestion (like a default). Anyone can make their own `List<CommandParser>` and configure it the way they want. This is just the default "right" way.